components/users/ui/src/main/resources/PhenoTips/PhenoTipsGroupSheet.xml

```diff
@@ -72,30 +72,35 @@ $xwiki.ssfx.use('icons/font-awesome/css/font-awesome.min.css')##
 (% class="xform" %)(((
 == $services.localization.render('phenotips.groups.manageMembers') ==
 
-(% class="buttonwrapper" %)[[{{icon name="wrench" /}} $services.localization.render('phenotips.groups.manageAdmins')&gt;&gt;path:$xwiki.getURL("${doc.fullName} Administrators", 'edit')||rel="__blank" class="button" style="float: right" id="manage-administrators"]]
+#set ($adminGroup = $xwiki.getDocument("${doc.fullName} Administrators"))
+#set ($postfix = " Managers")
+#if (!$adminGroup.isNew())
```

Comment:
Instead of loading the document, you can just do `#if (!$xwiki.exists("${doc.fullName} Administrators"))`